library/tabs/src/TabsWrapper.php

```diff
@@ -60,14 +60,27 @@ public function genCss()
         if (!defined('INCLUDED_TW_ONETIME_CSS')) {
             define(INCLUDED_TW_ONETIME_CSS, true);
             $s .= <<<EOD
-.ui-tabs .ui-tabs-panel {
-  padding: 0px 0px;
-}
-.ui-tabs .ui-tabs-nav .ui-tabs-anchor {
-  padding: .1em .4em;
-}
-EOD;
+        #enctabs-wrapper{
+          margin: 0px;
         }
+        .ui-tabs .ui-tabs-panel {
```

Comment:
Should PHP output be indented to reflect the way that the generating PHP code is indented? I would vote not. You don't need to change this, just a future topic for discussion.